examples/chart/teleport-kube-agent/README.md

```diff
@@ -5,6 +5,7 @@ with an existing Teleport cluster:
 - Teleport Kubernetes access
 - Teleport Application access
 - Teleport Database access
+- Teleport Kubernetes Discovery
```

Comment:
I think we might want to be careful here with the naming. We already have discovery that adds detects and adds Kubernetes clusters (EKS, GKE): https://goteleport.com/docs/kubernetes-access/discovery/ In its current state, the README suggests we support this feature and not the new one you are adding.

To avoid confusion, it would be handy to find a unique name for App and DB discovery in Kubernetes. I'm very bad at naming things but maybe we could differentiate:

- Teleport Kubernetes Cluster Discovery
- Teleport Kubernetes App Service Discovery

I guess this is more a product/marketing decision at this point 😅

Reply:
Yeah, it's been a problem for me during whole time, naming even in the code can get confusing 😅 
I think will change it to `Teleport Kubernetes App Discovery` for this PR, but we can try to find better name (encompassing app and DB) before the v14 release, it's just a text change, non-functional, so shouldn't be a problem.